azure/template.json

```
@@ -312,6 +312,14 @@
       "metadata": {
         "description": "Skylight auth Key."
       }
+    },
+
+    "cpu":{
```

```suggestion
    "dbServerCpuCores":{
```
also remove the blank line :316